impl/src/main/java/org/jboss/weld/bean/builtin/InstanceImpl.java

```diff
@@ -101,9 +101,17 @@ public T get() {
         if (bean != null) {
             return getBeanInstance(bean);
         } else if (isUnsatisfied()) {
-            throw BeanManagerLogger.LOG.unresolvableElement("Type: " + getType() + "; Qualifiers: " + getQualifiers());
+            throw BeanManagerLogger.LOG.injectionPointHasUnsatisfiedDependencies(ip,
```

Comment:
Including `ip.toString()` in the log message does not make sense here - it's a `DynamicLookupInjectionPoint` instance with no meaningful info.